src/octopeer-github/main/ElementSelectionBehaviour/ButtonElementSelectionBehaviour/AddEmoticonButtonElementSelectionBehaviour.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Created by Mathias on 2016-05-11.
+ * Class for selections the AddEmoticonButton
+ */
+class AddEmoticonButtonElementSelectionBehaviour extends ButtonElementSelectionBehaviour {
+
+    /**
+     * The description of the element, used to determine the element in the DOM-tree
+     * @type {string}
+     */
+    public elementDsc: string = ".js-merge-branch-action";
```

Comment:
If you looked carefully, all classes have this classname. These are merely placeholder for the DOM hooking